### PR TITLE
Social login: okdServiceLogin metatype initial drop

### PR DESCRIPTION
**Pull request by ayoho**

### CODE REVIEW

**Comment by ayoho** [comment]
#build

**Comment by andrea-pichler** [comment]
No message review requested from L2

**Comment by ayoho** [comment]
#build

File: dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -258,3 +261,9 @@ accessTokenSupported.desc=Determines whether to support access token authenticat
 
 accessTokenHeaderName=OAuth access token header name
 accessTokenHeaderName.desc=Name of the header to use when an OAuth access token is forwarded.
+
+userValidationApi=User validation API URL
+userValidationApi.desc=URL that is used to retrieve information about a user from a token.
+
+apiResponseCacheTime=User API response cache time
+apiResponseCacheTime.desc=The length of time for which the social login feature will cache the response from the user validation API. If a response is cached and the length of time that is specified by this attribute has passed, a new request to the user validation API is sent.
```
Comment (Rwalls1) [inline dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties]: Change "login feature will cache the response" to "login feature caches the response"

**Comment by ayoho** [comment]
#run-libby-bot

File: dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (ayoho) [reply dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties]: @Rwalls1 Good suggestion; just pushed the appropriate change. Let me know if there are any others you'd like to make.

**Pull request merged**